client/src/components/ReviewList.jsx

```diff
@@ -10,20 +10,63 @@ class ReviewList extends React.Component {
 		};
 	}
 	render() {
```

Comment:
For readability, could you leave an empty space in-between the constructor and render functions?